### DISCUSSION TITLE
Select option height reduction

### DISCUSSION THREAD

**Discussion by temitayoadeyeye** [react-hook-form/react-hook-form#2498] [Q&A]
I am new to react but I got a code in which react-hook-form was used and  a select function was imported so if i want to reduce the height of the select box, how can i make it slim as i have used style ={{height:'2.1rem'}} to reduce within the form.control tag but within the controller tag it is not working, how can i reduce this box 

![select](https://user-images.githubusercontent.com/69305577/89527629-8cea4c00-d7e1-11ea-8793-d89a62fa9958.png)

**Comment by bluebill1049** [comment]
hmm this is not a hook form related question, could you ask in react-select channel.

**Reply by temitayoadeyeye** [reply]
Okay. Thanks
